spec/03-plugins/19-acl/01-api_spec.lua

```diff
@@ -263,4 +263,187 @@ describe("Plugin: acl (API)", function()
       end)
     end)
   end)
+
+  describe("/acls/", function()
+
+    after_each(function()
+      helpers.dao:truncate_table("acls")
+    end)
+
+    describe("POST", function()
+      it("creates an ACL group", function()
+        local res = assert(admin_client:send {
+          method = "POST",
+          path = "/acls",
+          body = {
+            group = "hello-group",
+            consumer_id = consumer.id
+          },
+          headers = {
+            ["Content-Type"] = "application/json"
+          }
+        })
+        local body = cjson.decode(assert.res_status(201, res))
+        assert.equal(consumer.id, body.consumer_id)
+        assert.equal("hello-group", body.group)
+      end)
+      describe("errors", function()
+        it("returns bad request", function()
+          local res = assert(admin_client:send {
+            method = "POST",
+            path = "/acls",
+            body = {},
+            headers = {
+              ["Content-Type"] = "application/json"
+            }
+          })
+          local body = assert.res_status(400, res)
+          assert.equal([[{"consumer_id":"consumer_id is required","group":"group is required"}]], body)
```

Comment:
it might make more sense to test table equality directly, by decoding `body`, rather than a string representation of the JSON? table keys are not encoded deterministically (in our environment, to my knowledge), so this test can fail at random.